components/breadcrumbs/test/breadcrumb.test.js

```diff
@@ -0,0 +1,66 @@
+import '../breadcrumb.js';
+import { expect, fixture, html } from '@open-wc/testing';
+import { runConstructor } from '../../../tools/constructor-test-helper.js';
+
+describe('d2l-breadcrumb', () => {
+
+	let elem;
+	beforeEach(async() => {
+		elem = await fixture(html`
+			<d2l-breadcrumb></d2l-breadcrumb>
+		`);
+	});
+
+	describe('attribute reflection', () => {
+		it('should reflect "href" property to attribute', async() => {
+			elem.href = '/someurl';
+			await elem.updateComplete;
+			expect(elem.getAttribute('href')).to.equal('/someurl');
+		});
+
+		it('should reflect "text" property to attribute', async() => {
+			elem.text = 'Awesome Url';
+			await elem.updateComplete;
+			expect(elem.getAttribute('text')).to.equal('Awesome Url');
+		});
+
+		it('should reflect "target" property to attribute', async() => {
+			elem.target = 'Awesome Target';
+			await elem.updateComplete;
+			expect(elem.getAttribute('target')).to.equal('Awesome Target');
+		});
+
+		it('should reflect "ariaLabel" property to attribute', async() => {
+			elem.ariaLabel = 'Awesome Label';
+			await elem.updateComplete;
+			expect(elem.getAttribute('aria-label')).to.equal('Awesome Label');
+		});
+
+		it('should reflect "_compact" property to attribute', async() => {
+			elem._compact = true;
+			await elem.updateComplete;
+			expect(elem.hasAttribute('compact')).to.be.true;
+		});
+	});
+
+	describe('constructor', () => {
+		it('should construct', () => {
```

Comment:
Can you add a constructor test for `d2l-breadcrumbs` too?

Reply:
Sure. will do

Reply:
@AllanKerr it already exists under `describe('basic attribute tests'` . Should I moved it out?

Reply:
Sorry, I completely missed that! 😳 Normally I'd put it in a higher up `describe` block but thats super nit-picky so your call.

Reply:
yeah let me move that out then.

Reply:
Done ✔️